packages/mui-codemod/src/v5.0.0/joy-text-field-to-input.test/expected.js

```diff
@@ -0,0 +1,8 @@
+// the codemod should transform only Joy TextField
+import { Input as JoyTextField } from "@mui/joy";
+import JoyTextField2 from "@mui/joy/Input";
+
+<div>
+  <JoyTextField />
```

Comment:
Can this be transformed to `<JoyInput />`?

Reply:
Agree, done in these commits: https://github.com/mui/material-ui/pull/35462/commits/184fa6b04dc09de53f6a503c834e338f00044102 and https://github.com/mui/material-ui/pull/35462/commits/8ca049bfc53265635b8685baabd8f93561e63cc1